### ISSUE TITLE
[Bug]: Weird output after executing 'postinstall' script

### ISSUE THREAD

**Issue by stillwu** [nuxtrdev/nuxtr-vscode#14]
### Environment

RootDir: D:/project/my-nuxt-app                                                                                                                 12:45:50
Nuxt project info:                                                                                                                              12:45:50

------------------------------
- Operating System: Windows_NT
- Node Version:     v20.2.0
- Nuxt Version:     3.5.2
- Nitro Version:    2.4.1
- Package Manager:  pnpm@8.6.2
- Builder:          vite
- User Config:      devtools, modules
- Runtime Modules:  @nuxtjs/tailwindcss@6.7.2, nuxt-icon@0.4.1
- Build Modules:    -
------------------------------

### What went wrong?

I installed some modules, and I executed **'postinstall'** script with **Nuxtr** (click 'postinstall' button in Actions). Then **Nuxtr** outputs some infos in VS Code output bar, but it seems to occur character encoding error. I don't know if it should be like this. Here comes the infos:

```bash
> nuxt-app@ postinstall D:\project\my-nuxt-app
> nuxt prepare

[90mNuxi [1m3.5.2[22m[39m
[90m[[90mnuxt:tailwindcss[90m][39m [36mi[39m Using default Tailwind CSS file
[32m√[39m Types generated in .nuxt
```

Screenshot:

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/5f0c4b8c-d311-4504-b532-14519e33c782)

My editor's charset is UTF-8

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/acb6d8ed-407d-4dec-af8f-3d27160d0c26)

Should it be displayed like this format? Or it will be improved.

### How to reproduce it?

click 'postinstall' script button in Actions.

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/5972323c-63da-4c7a-b190-c4af32e40a1c)

**Comment by adhamfarrag** [comment]
Hello @whoeverimf5 

Two questions:

1. Do you get the same bug when you do the command from Terminal ? 
2. If yes, I tried to debug the issue on a Windows machine and the solution was to use Node `18.15.0`. 

Let me know.

**Comment by whoeverimf5** [comment]
> Hello @whoeverimf5
> 
> Two questions:
> 
> 1. Do you get the same bug when you do the command from Terminal ?
> 2. If yes, I tried to debug the issue on a Windows machine and the solution was to use Node `18.15.0`.
> 
> Let me know.

1. No, it would display normally

**OUTPUT** in terminal:

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/59e5d020-cd16-4b3c-b2cb-147cf1cd14ae)

**OUTPUT** in Nuxtr:

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/79c7b7c5-1630-45b1-87dd-79e4265b2c12)

**Comment by adhamfarrag** [comment]
Can you do it again and share the error you see in VSCode console ?

**Comment by whoeverimf5** [comment]
I updated Nuxtr just now, uh... I get the same error.

```bash

2023-06-15 15:43:29.691 [info] pnpm outdated --json
2023-06-15 15:43:34.465 [info] 
> nuxt-app@ postinstall D:\project\my-nuxt-app
> nuxt prepare

[90mNuxi [1m3.5.2[22m[39m
[90m[[90mnuxt:tailwindcss[90m][39m [36mi[39m Using default Tailwind CSS file
[32m√[39m Types generated in .nuxt

```

screenshot:

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/e73dba27-366e-4edd-8b2d-5eb38a9894b0)

**Comment by adhamfarrag** [comment]
Yeah, the last release wasn't tackling your issue. 

VSCode Console can be oepend from Help > Toggle Developer Tools. 
Please, do the command again and share what you see.

**Comment by whoeverimf5** [comment]
Ok, the error is:

```bash

[Extension Host] Error: Command failed: pnpm outdated --json

	at ChildProcess.exithandler (node:child_process:409:12)
	at ChildProcess.emit (node:events:513:28)
	at maybeClose (node:internal/child_process:1121:16)
	at ChildProcess._handle.onexit (node:internal/child_process:304:5)

```

This error occurs when I open Nuxtr or I click 'Focus on Project View'.

screenshot:

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/76519998/52fd3c0b-e1e6-42de-8407-2b9c9bf359d9)

**Comment by adhamfarrag** [comment]
Weird. what's the version if your PNPM version ?

**Comment by whoeverimf5** [comment]
> Weird. what's the version if your PNPM version ?

8.6.2

**Comment by adhamfarrag** [comment]
@stillwu Fixed in version 0.1.7. Please update :)
